package.json

```diff
@@ -112,51 +109,51 @@
     "eslint-plugin-jsx-a11y": "5.1.1",
     "eslint-plugin-react": "7.11.1",
     "file-loader": "2.0.0",
-    "flow-bin": "0.85.0",
-    "flow-coverage-report": "0.6.0",
+    "flow-bin": "0.95.1",
+    "flow-coverage-report": "0.6.1",
     "fs-extra": "7.0.1",
     "html-webpack-plugin": "4.0.0-beta.4",
     "husky": "0.14.3",
     "identity-obj-proxy": "3.0.0",
     "ip": "1.1.5",
     "jasmine-core": "3.3.0",
-    "jest": "23.6.0",
-    "jest-environment-enzyme": "7.0.1",
-    "jest-enzyme": "7.0.1",
-    "lint-staged": "7.2.2",
+    "jest": "24.5.0",
+    "jest-environment-enzyme": "7.0.2",
+    "jest-enzyme": "7.0.2",
+    "lint-staged": "8.1.5",
     "mini-css-extract-plugin": "0.4.5",
-    "node-sass": "4.5.3",
-    "postcss-flexbugs-fixes": "3.2.0",
+    "node-sass": "4.11.0",
+    "postcss-flexbugs-fixes": "4.1.0",
     "postcss-loader": "3.0.0",
-    "react-app-polyfill": "0.1.3",
-    "react-dev-utils": "6.1.1",
+    "react-app-polyfill": "0.2.2",
+    "react-dev-utils": "8.0.0",
     "react-error-overlay": "1.0.10",
-    "react-test-renderer": "15.6.2",
+    "react-test-renderer": "16.8.4",
     "recompose": "0.30.0",
```

Comment:
do we need `recompose`?

Reply:
Recompose in the devDependences, it can be used with storybook